web/src/features/moreCast2/components/forecastCell.test.tsx

```diff
@@ -0,0 +1,160 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import ForecastCell from 'features/moreCast2/components/ForecastCell'
+import { GridRenderCellParams } from '@mui/x-data-grid'
+
+const params: Pick<GridRenderCellParams, 'row' | 'formattedValue'> = {
+  row: undefined,
+  formattedValue: '1'
+}
+
+describe('ForecastCell', () => {
+  it('should have input disabled when disabled prop is true', () => {
+    const { container } = render(
+      <ForecastCell
+        disabled={true}
+        label="foo"
+        showGreaterThan={false}
+        showLessThan={false}
+        value={params.formattedValue}
+      />
+    )
+
+    const inputElement = container.querySelector('input')
+    expect(inputElement).toBeInTheDocument()
+    expect(inputElement).toBeDisabled()
+  })
+  it('should have input enabled when disabled prop is false', () => {
+    const { container } = render(
+      <ForecastCell
+        disabled={false}
+        label="foo"
+        showGreaterThan={false}
+        showLessThan={false}
+        value={params.formattedValue}
+      />
+    )
+
+    const inputElement = container.querySelector('input')
+    expect(inputElement).toBeInTheDocument()
+    expect(inputElement).toBeEnabled()
+  })
+  it('should show less than icon when showLessThan is true', () => {
+    const { queryByTestId } = render(
+      <ForecastCell
+        disabled={false}
+        label="foo"
+        showGreaterThan={false}
+        showLessThan={true}
+        value={params.formattedValue}
+      />
+    )
+
+    const element = queryByTestId('forecast-cell-less-than-icon')
+    expect(element).toBeInTheDocument()
+  })
+  it('should not show less than icon when showLessThan is false', () => {
+    const { queryByTestId } = render(
+      <ForecastCell
+        disabled={false}
+        label="foo"
+        showGreaterThan={false}
+        showLessThan={false}
+        value={params.formattedValue}
+      />
+    )
+
+    const element = queryByTestId('forecast-cell-less-than-icon')
+    expect(element).not.toBeInTheDocument()
+  })
+  it('should show greater than icon when showGreaterThan is true', () => {
```

Comment:
What if `showGreaterThan` and `showLessThan` are both true?

Reply:
Technically both icons would display which I could test for, but logic in the parent component prevents this from happening.

```      showLessThan = formattedValue < actualValue
      showGreaterThan = formattedValue > actualValue
```

Any thoughts on what we should do/test?

Reply:
I think we just need to show one or the other, so we could just use one boolean input, but we also need to consider the unlikely equal case though, which is what is achieved when both booleans are false. 

I would say disregard if the parent guarantees one or both are false.

Reply:
A single boolean won't quite cover it as no icon is rendered when forecast === actual. I suppose we could pass both the forecast and actual values as props but the component doesn't really need to know about both values as it only displays the forecast value if it exists.

I've added one more test to explicitly cover the case where `showLessThan` and `showGreaterThan` are both false.